COPYRIGHT

```diff
@@ -0,0 +1,7 @@
+All role code is Copyright 2023 by Jeff Geerling.
+
+This role is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+
+This role is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with Foobar. If not, see <https://www.gnu.org/licenses/>.
```

Comment:
I believe you are supposed to replace "Foobar" with the name of the package in this line.

Reply:
Fixed!